### PR TITLE
add tests for invalid cases when creating a collection

### PR DESCRIPTION
**Pull request by cristian0995**

### CODE REVIEW

File: pages/desktop/frontend/collections.py [hunk 1]
```diff
@@ -118,6 +118,8 @@ class CollectionCreate(Region):
             '.EditableCollectionAddon',
         )
 
+        _warning_text_locator = (By.CSS_SELECTOR, '.Notice-error > * > * >.Notice-text')
```
Comment (jrbenny35) [inline pages/desktop/frontend/collections.py:121]: Could you not just use ```.Notice-text```?

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Looks great, just 1 question.

File: tests/frontend/test_collections.py [hunk 2]
```diff
@@ -457,3 +457,63 @@ def test_collection_sort_addons_by_popularity(selenium, base_url, variables):
...
+    collections.create.save_collection()
+    collections.collection_detail.wait_for_details_to_load()
+
+    collections.driver.get('https://addons.allizom.org/en-US/firefox/collections/')
+    collections.click_create_collection()
+    collections.create.set_name('def')
+    collections.create.slug_label_element.clear()
+    collections.create.set_slug('abc')
+    collections.create.save_collection()
+    assert 'This custom URL is already in use by another one of your collections.' in collections.create.warning_text
```
Comment (AlexandraMoga) [inline tests/frontend/test_collections.py:508]: let's add long strings like these in the variables file (in this case should be the `stage.json `file)

File: tests/frontend/test_collections.py [hunk 3]
```diff
@@ -457,3 +457,63 @@ def test_collection_sort_addons_by_popularity(selenium, base_url, variables):
...
+def test_create_collection_with_already_used_url(selenium, base_url):
+    collections = Collections(selenium, base_url).open().wait_for_page_to_load()
+    collections.login('collection_user')
+    collections.click_create_collection()
+    collections.create.set_name('abc')
+    collections.create.slug_label_element.clear()
+    collections.create.set_slug('abc')
+    collections.create.save_collection()
+    collections.collection_detail.wait_for_details_to_load()
+
```
Comment (AlexandraMoga) [inline tests/frontend/test_collections.py:501]: we do not need this empty line here, but you could add a comment to better explain what the next bit of this test is supposed to do

**Review by AlexandraMoga — CHANGES_REQUESTED** [review]

**Comment by AlexandraMoga** [comment]
I've added a few change requests as well :stuck_out_tongue_winking_eye:

File: pages/desktop/frontend/collections.py (hunk 1 above)
Comment (cristian0995) [reply pages/desktop/frontend/collections.py:121]: Just '.Notice-text' will find me the yellow notice too. 
I change it to '.Notice-error .Notice-text' :)

File: tests/frontend/test_collections.py (hunk 2 above)
Comment (cristian0995) [reply tests/frontend/test_collections.py:508]: Ok, i added 'collection_reused_url_warning' to stage.json

File: tests/frontend/test_collections.py [hunk 4]
```diff
@@ -457,3 +457,63 @@ def test_collection_sort_addons_by_popularity(selenium, base_url, variables):
...
+@pytest.mark.nondestructive
+def test_create_collection_with_invalid_symbols_in_url(selenium, base_url):
+    collections = Collections(selenium, base_url).open().wait_for_page_to_load()
+    collections.login('collection_user')
+    collections.click_create_collection()
+    collections.create.set_name('abc')
+    collections.create.set_slug('^_^')
+    collections.create.save_collection()
+    assert 'The custom URL must consist of letters, numbers, underscores or hyphens.' in collections.create.warning_text
```
[the remark this answers is not in the document]
Comment (cristian0995) [reply tests/frontend/test_collections.py:519]: Ok, I added 'collection_invalid_custom_url_warning' to stage.json

File: tests/frontend/test_collections.py (hunk 3 above)
Comment (cristian0995) [reply tests/frontend/test_collections.py:501]: Done :)

**Review by AlexandraMoga — APPROVED** [review]

**Pull request merged**